ui/window.go

```diff
@@ -1772,11 +1783,17 @@ func (window *Window) handleGlobalShortcuts(event *tcell.EventKey) *tcell.EventK
 		buttonBar.AddItem(tview.NewBox(), 1, 0, false)
 		buttonBar.AddItem(exitButton, 0, 1, false)
 
+		descriptionBar := tview.NewFlex()
+		descriptionBar.SetDirection(tview.FlexColumn)
+
+		descriptionBar.AddItem(shortcutDescription, 0, 1, false)
+
```

Comment:
The description bar is not necessary, since it's just a container wrapped around a single item. Instead add the item to `shortcutsView` directly.